docs/contributing.rst

```diff
@@ -5,11 +5,29 @@ Contributing
 Contributions are welcome, and they are greatly appreciated! Every little bit
 helps, and credit will always be given.
 
-You can contribute in many ways:
+Getting in touch
+----------------
+
+Join the `Ansible Forum <https://forum.ansible.com>`_, the default communication platform for questions and help, development discussions, events, and much more.
+`Register <https://forum.ansible.com/signup?>`_ to join the community.
+Search by categories and tags to find interesting topics or start a new one; subscribe only to topics you need!
+
+* `Get Help <https://forum.ansible.com/c/help/6>`_: get help or help others. Please add appropriate tags if you start new discussions, for example `ansible-rulebook`.
+* `Posts tagged with 'ansible-rulebook' <https://forum.ansible.com/tag/ansible-rulebook>`_: subscribe to participate in project-related conversations.
+* `Bullhorn newsletter <https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn>`_: used to announce releases and important changes.
+* `Social Spaces <https://forum.ansible.com/c/chat/4>`_: gather and interact with fellow enthusiasts.
+* `News & Announcements <https://forum.ansible.com/c/news/5>`_: track project-wide announcements including social events.
+
+See `Navigating the Ansible forum <https://forum.ansible.com/t/navigating-the-ansible-forum-tags-categories-and-concepts/39>`_ for some practical advice on finding your way around.
+
+You can also check the `Matrix chat <https://matrix.to/#/#eda:ansible.com>`__, or via the
+event-driven-automation@redhat.com email.
```

Comment:
```suggestion
You can also check the `Matrix chat <https://matrix.to/#/#eda:ansible.com>`__, or via the
event-driven-automation@redhat.com email.
```
i'm not sure maybe we shouldn't offer any communication via emails here? i think it's OK for reporting security things or CoC violation things but there are special emails for them.

Reply:
I feel like this is a project team decision so I've opened https://github.com/ansible/ansible-rulebook/issues/714

If @Alex-Izquierdo is OK with removing the email here, that's fine too, and we can close the issue.

Reply:
@oraNod The email is intentional, anyway @josephpisciotta can confirm it. 
But I think we should remove the mention of the matrix chat in favor of the forum as [it has been done in eda-server](https://github.com/ansible/eda-server/pull/1024/files#diff-419d79ae7d440d49b3fa5fa7739f5f3a359e3ca42c3fa942e5604ae3474eb35bR34).

Reply:
Sheesh. I guess I should be consistent with my own PRs... What can I say? It was early in the morning and my caffeine dependency was not loaded. :coffee: